otherlibs/stdune/src/wait4_stubs.c

```diff
@@ -56,11 +56,11 @@ value dune_wait3(value flags) {
   struct rusage ru;
 
   caml_enter_blocking_section();
-  pid = wait3(&status, cv_flags, &ru);
+  pid = wait4(-1, &status, cv_flags, &ru);
```

Comment:
Instead of `-1,` how about adding this as an argument to `dune_wait4` and propagating it everywhere?

Reply:
Ah, hm, I tried using the `Pid.t`, I think defined [here](https://github.com/ocaml/dune/blob/b326c30331b5fdb039e06b02b88bdff96bf17da3/otherlibs/stdune/src/pid.ml#L8-L11), but when converting from `Pid.of_int -1` (`-1` being any child process), this would end up failing. I guess two options are, change the implementation of the `pid.ml` file be changed to allow `-1`, or change the type signatures to accept a `int` instead?